components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -813,13 +813,11 @@ func prepareIDELaunch(cfg *Config, ideConfig *IDEConfig) *exec.Cmd {
 	if ideConfig.Entrypoint == "/ide/startup.sh" && len(args) == 0 {
 		args = append(args, "{WORKSPACEROOT}")
 		args = append(args, "--port", "{IDEPORT}")
-		args = append(args, "--hostname", "{IDEHOSTNAME}")
```

Comment:
@filiptronicek @jeanp413 You tested that it does not affect stable version?

Reply:
@akosyakov yeah it's fine as it's not a valid argument